src/engine/input/key_mapping/SpecifiedInputKey.cs

```diff
@@ -194,10 +194,41 @@ public static (JoyButton Code, int Device) UnpackCodeAndDevice(ulong packed)
         return ((JoyButton)code, device);
     }
 
-    public Control GenerateGraphicalRepresentation()
+    public Control GenerateGraphicalRepresentation(LabelSettings labelSettings)
     {
         var container = new HBoxContainer();
 
+        if (Shift || Control || Alt)
```

Comment:
I think this should only be done if the event type is not a controller button, as an input like "shifit + controller button" doesn't make sense (and hopefully is prevented by the binding system from being generated).